### ISSUE TITLE
Inventory Bug

### ISSUE THREAD

**Issue by ghost** [Crazy-Crew/Crazy-Auctions#101]
Hi, today i just downloaded the plugin and put the italian default config. and i don't why i can't get block in gmc.... you can but you need to click more times to the item, and it wasn't possible to clear the inv. bar....

**Comment by badbones69** [comment]
Could you show me a gif/video of this?

**Comment by ooULuckyoo** [comment]
Hello, it appears that I'm having the same issue so I decided to comment here instead of opening a new ticket. This is the link for the video of my issue: https://youtu.be/fkFAJWwfRL0

Here is the Inventory Issue when having CrazyAuctions installed.

1. Attempted middle-clicking to get the block.

2. Attempted to drag-and-drop item into hotbar.

3. I am able to press 1-9 to 'quick select' the item into my hotbar slot after pressing multiple times (about 4-5).

4. After selecting items, I attempted to throw items onto the ground, they disappear but don't drop at all.

Thanks, I hope this gets resolved soon.
ULucky
